Sources/Testing/Traits/Tag.Color+Loading.swift

```diff
@@ -36,13 +36,15 @@ private var _homeDirectoryPath: String? {
 /// The path to the current user's App Data directory, if known.
 private var _appDataDirectoryPath: String? {
   var appDataDirectoryPath: PWSTR? = nil
-  if S_OK == SHGetKnownFolderPath(FOLDERID_LocalAppData, 0, NULL, &appDataDirectoryPath), let appDataDirectoryPath {
-    defer {
-      CoTaskMemFree(appDataDirectoryPath)
+  return withUnsafePointeR(to: FOLDERID_LocalAppData) {
```

Comment:
Well _now_ who spelled it incorrectly. ;)

If we need a pointer to the folder ID, can we just use `&`? (If not, please label the closure argument to match house style.)

Reply:
Haha, that is what happens when you test and commit across different environments!

Reply:
We could copy it to a local:

```swift
var FOLDERID_LocalAppData = FOLDERID_LocalAppData
if S_OK == SHGetKnownFolderPath(&FOLDERID_LocalAppData, 0, nil, &appDataDirectoryPath), let appDataDirectoryPath {
```

Reply:
That sounds good to me.